packages/aws-cdk/lib/api/bootstrap/bootstrap-template.yaml

```diff
@@ -235,6 +235,24 @@ Resources:
     Type: AWS::ECR::Repository
     Properties:
       ImageTagMutability: IMMUTABLE
+      # Untagged images should never exist but Security Hub wants this rule to exist
+      LifecyclePolicy:
+        LifecyclePolicyText: |
+          {
+            "rules": [
+              {
+                "rulePriority": 1,
+                "description": "no-op",
```

Comment:
Can we not call this `no-op`, this will just confuse people. Something like "Untagged images should not exist, but expire any older than 1 year" should do